src/smm/meson.build

```diff
@@ -0,0 +1,28 @@
+if conf.get('ENABLE_OPENSSL')
+    libsmm = library('smm-local',
+        version: '11.0.2',
+        include_directories: core_inc,
+        sources: files(
+            'sm-card-authentic.c',
+            'sm-card-iasecc.c',
+            'sm-cwa14890.c',
+            'sm-global-platform.c',
+            'smm-local.c'
+        ),
+        link_with: [
+            libopensc,
+            libsm
+        ],
+        dependencies: depopenssl,
+        install: true
+    )
+
+    modpkgconfig.generate(
+        libsmm,
+        name: 'OpenSC smartcard framework',
+        version: meson.project_version(),
+        description: 'OpenSC Secure Messaging module',
+        url: 'https://github.com/OpenSC/OpenSC',
+        filebase: 'opensc-smm'
+    )
```

Comment:
Does this install the pkgconfig file for the sm? I think this is just internal for opensc.

Reply:
This library is not used internally and should not be used externally. Why to compile it at all?

Reply:
Thanks. Fixed locally.